velox/dwio/common/compression/Compression.h

```diff
@@ -60,37 +67,62 @@ class Decompressor {
   const std::string streamDebugInfo_;
 };
 
+struct CompressionOptions {
+  /// Format specific compression/decompression options
+  union Format {
+    struct {
+      /// Window bits determines the history buffer size and whether
+      /// header/trailer is added to the compression block.
+      int windowBits;
+      /// Compression level determines the compression ratio. Zlib supports
+      /// values ranging from 0 (no compression) to 9 (max compression)
+      int32_t compressionLevel;
+    } zlib;
+
+    struct {
+      int32_t compressionLevel;
+    } zstd;
+  } format;
+
+  uint32_t compressionThreshold;
+};
+
 /**
  * Create a decompressor for the given compression kind.
  * @param kind the compression type to implement
```

Comment:
Can you please use Upper case for the first characters? E.g. 
@param kind The compression type to implement

ditto